rsp_vision/console_application/app.py

```diff
@@ -45,8 +50,16 @@ def analysis_pipeline() -> None:
 
     CLI or GUI functionality is added here.
     """
-    # pipeline draft
-    start_logging()
+    config = read_config_file(config_path)
+    swc_blueprint_spec = SWC_Blueprint_Spec(
+        project_name="rsp_vision",
+        raw_data=False,
+        derivatives=True,
+        local_path=Path(config["paths"]["output"]),
```

Comment:
Now that you also use the output path, you should update the repo README accordingly. There is still says "The only path that matters at this stage is the allen_dff path, which should point to the folder where you stored the pre-processed data." Now the output path also matters.